internal/microsoft_edge.py

```diff
@@ -68,8 +70,18 @@ def prepare(self, job, task):
         self.bodies_path = os.path.join(task['dir'], 'bodies')
         if not os.path.isdir(self.bodies_path):
             os.makedirs(self.bodies_path)
-        if not task['cached']:
-            self.clear_cache()
```

Comment:
Sorry to keep dragging this out.  Can you also leave the explicit clear in place as both a backup and to catch anything that the reg key may not clear?